### ISSUE TITLE
Programming language(s) used in repository detected by Linguist is incorrect

### ISSUE THREAD

**Issue by bryce-carson** [ashokkrish/episim#151]
https://github.com/github-linguist/linguist/blob/master/docs/how-linguist-works.md

**Comment by ashokkrish** [comment]
I really home we can somehow make it not look like a Javascript project and switch to the primary programming language.

**Comment by bryce-carson** [comment]
> I really hope we can somehow make it not look like a Javascript project and switch to the primary programming language.

Yeah, we can. It just takes effort. I'm not sure why it is detected incorrectly, but apparently linguist is only seeing a couple files! I followed GitHub's documentation to correct this but didn't have any better luck using their tool locally to generate a "linguist file" which would override detection on their servers and use our file instead.

At the moment, our best bet is to ask someone on StackOverflow to help us by submitting a pull request to fix the issue, or give us fool-proof instructions.
